drivers/usb/gadget/Kconfig

```diff
@@ -485,12 +485,12 @@ config USB_CONFIGFS_F_TCM
 	  Both protocols can work on USB2.0 and USB3.0.
 	  UAS utilizes the USB 3.0 feature called streams support.
 
-config USB_CONFIGFS_F_XAPTUM_PSOCK
-        bool "Xaptum psock driver"
+config USB_CONFIGFS_F_XAPTUM_SCM
+        bool "Xaptum SCM driver"
 	depends on USB_CONFIGFS
-	select USB_F_XAPTUM_PSOCK
+	select USB_F_XAPTUM_SCM
         help
-          This driver adds the psock type to the Linux kernel for use
+          This driver adds the SCM function to the Linux kernel for use
```

Comment:
Let's make this comment more specific too.  How about (borrowing from the SCM protocol document)

```
The Socket Control Model is protocol for a USB device to manage 
and use Berkeley/POSIX-style sockets on the host.  It allows the
device to communicate with remote servers using the network
connection of the host.  Currently supported protocols include
TCP/IPv4 and TCP/IPv6.
```